metadata.yml

```diff
@@ -19,3 +22,9 @@ annotations:
   io.kubewarden.policy.url: https://github.com/yourorg/{{project-name}}
   io.kubewarden.policy.source: https://github.com/yourorg/{{project-name}}
   io.kubewarden.policy.license: Apache-2.0
+  # The next two annotations are used in the policy report generated by the
+  # Audit scanner. Severity indicates policy check result criticality and
+  # Category indicates policy category. See more here:
+  # https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html
```

Comment:
I'm not super happy pointing to docs that we don't control tbh. Maybe we should substitute it for `see more on docs.kubewarden.io`. And there we can have an extract of those policy-report docs, the different severities, that policy administrators can overwrite the values, and that we follow the convention of not setting severity `critical` by default, so policy administrators can use it for emergencies.